.github/workflows/modmesh_viewer.yml

```diff
@@ -0,0 +1,95 @@
+name: modmesh_viewer
+
+on:
+  push:
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+        matrix:
+          os: [ubuntu-22.04, macos-11]
```

Comment:
`macos-11` does not work: https://github.com/tai271828/modmesh/runs/6560081058?check_suite_focus=true#step:7:70 .